### ISSUE TITLE
point users to android-dev

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#269]
as soon as the [single-folder milestone](https://github.com/deltachat/deltachat-core/milestone/2) gets merged into master and this version is used for deltachat-desktop, users that want to use a multi-client-setup with android and desktop should be pointed to use android-dev.

the reason for this as the current "stable" android-f-droid version moved messages from the INBOX to the DeltaChat folder that is no longer watched by single-folder-deltachat-desktop then.

android-dev is available at https://github.com/deltachat/deltachat-android-ii/releases and already uses the single-folder branch today.

**Comment by ralphtheninja** [comment]
I was thinking we could have some sorts of "release banner" at the top with a X-button to dismiss?

**Comment by r10s** [comment]
> I was thinking we could have some sorts of "release banner" at the top with a X-button to dismiss?

whatever is not too complicated ;)

**Comment by r10s** [comment]
just merged the single-folder milestone to master, the next released version will contain this change then. (@ralphtheninja afaik node only uses tagged versions)

**Comment by karissa** [comment]
@r10s yes, deltachat-node is in a beta state (`0.24.1`) so we can publish the single-folder milestone to `0.25` and it won't get used on desktop until desktop is ready and manually upgrades to `0.25.x`

**Comment by r10s** [comment]
@karissa cool - i'd like to play around first here a bit with the "staged" commits and will release 0.25 then probably later this week. i think, it's not urgent in any way.

**Comment by r10s** [comment]
0.25.0 is just released. 
so, if you use this new core, it would be great to point multi-device-users to android-dev
